src/app/components/Footer/index.tsx

```diff
@@ -64,6 +70,9 @@ export default () => {
         }
       >
         <List elements={elements} trustProjectLink={trustProjectLink} />
+        {collectiveNewsroomText && (
```

Comment:
Let's include an `!isLive()` check here, so that we can merge the PR and perform accessibility tests on the test environment

Reply:
added :)